.github/pull_request_template.md

```diff
@@ -0,0 +1,36 @@
+## Description / Purpose
+
+Please include a description of the purpose and approach of this pull request's commits.
+This area should provide an overview of why the work is happening and, within reason, should include relevant links to the ticket, product/design briefs, and engineering plans. We shouldn’t assume familiarity with the history of this work. Please also include relevant motivation and context. List any dependencies that are required for this change.
+
+## Types of changes
+
+What types of changes does your code ?
+_Put an `x` in the boxes that apply_
+
+- [ ] Bugfix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
```

Comment:
@iancmx I think in our `Contributing` doc we wrote to email development@procurify.com if you want to introduce a breaking change. Should we change this or change our `Contributing` document?

Reply:
I believe that we should change this, I'll modify this checkmark to prompt the dev to email development@procurify.com if the dev wants to introduce a breaking change